crates/ryot_assets/src/core/tests.rs

```diff
@@ -113,26 +114,11 @@ fn sprite_sheet_set_fixture() -> SpriteSheetDataSet {
         }),
     ];
 
-    SpriteSheetDataSet::from_content(&vec)
+    (&*vec).into()
```

Comment:
ew

Reply:
okok, I've removed all the usages of &* from the codebase hahaha